src/transformers/models/owlvit/feature_extraction_owlvit.py

```diff
@@ -85,6 +85,13 @@ def __init__(
         image_std=None,
         **kwargs
     ):
+        # Early versions of the OWL-ViT config on the hub had "rescale" as a flag. This clashes with the
+        # vision feature extractor method `rescale` as it would be set as an attribute during the super().__init__
+        # call. This is for backwards compatibility.
+        if "rescale" in kwargs:
+            rescale_val = kwargs.pop("rescale")
+            kwargs["do_rescale"] = rescale_val
+
```

Comment:
Looks good! Will we eventually remove this part?

Reply:
Yes. I think we can leave it for a short period of time so that if anyone pushes to the hub and updates the model, pushing the feature extractor with it, then the `rescale` parameter will be renamed in the `preprocessor_config.json`. After that, we could add a warning if `rescale` is in detected in the config - asking the person to update their config as it will no longer be supported as a flag in the future.